### ISSUE TITLE
build on a clean Raspberry PI fails using the 'one script' method

### ISSUE THREAD

**Issue by DuaneEllis-TI** [openthread/borderrouter#18]
Using a FRESH raspbian image, on a RaspberryPI 3 B+ using this image:
```
$ sha1sum.exe 2017-04-10-raspbian-jessie-lite.*
dad2ba8a228f14e64af7d80e1fd2503facb70c61 *2017-04-10-raspbian-jessie-lite.img
c24a4c7dd1a5957f303193fee712d0d2c0c6372d *2017-04-10-raspbian-jessie-lite.zip
```
The error message I get is this (in contrast, the Beagle Bone Black) goes past this step.
perhaps, when these docs where written the NodeJS package was already installed?
And thus the script skipped over this part?
```
## Installing the NodeSource Node.js v6.x repo...

## You appear to be running on ARMv6 hardware. Unfortunately this is not currently supported by 
the NodeSource Linux distributions. Please use the 'linux-armv6l' binary tarballs available
directly from nodejs.org for Node.js v4 and later.

pi@raspberrypi:~/borderrouter$
```

**Comment by bukepo** [comment]
I think #24 will solve this issue.

**Comment by DuaneEllis-TI** [comment]
How do you think #24 will solve this?  I do not see a change to the OneScript files that are giving the error.

**Comment by bukepo** [comment]
I'm sorry, I think we should fix the issue in that PR. @LeZhang2016 could you please remove line 58~64 in [script/bootstrap](https://github.com/openthread/borderrouter/blob/master/script/bootstrap) in #24

**Comment by jwhui** [comment]
Resolved by #24.
